app/src/main/java/org/thoughtcrime/securesms/onboarding/loading/LoadingActivity.kt

```diff
@@ -40,11 +37,12 @@ class LoadingActivity: BaseActionBarActivity() {
     private fun register(skipped: Boolean) {
         prefs.setLastConfigurationSyncTime(System.currentTimeMillis())
 
-        val flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
-
         when {
-            skipped -> startPickDisplayNameActivity(true, flags)
-            else -> startMessageNotificationsActivity(flags)
+            skipped -> startPickDisplayNameActivity(
```

Comment:
Does the word 'skipped' make sense? That makes it sound user driven, but it really was a timeout.

Reply:
Renamed load failed, because I guess we could have other reasons.